migration_util.py

```diff
@@ -1,26 +1,23 @@
 from __future__ import print_function
-from apiclient.discovery import build
+from googleapiclient.discovery import build
+#from apiclient.discovery import build
 from httplib2 import Http
 from oauth2client import file, client, tools
 import requests
 import urllib
 from io import BytesIO
 from pathlib import Path
 
+
 def authorize_with_google():
-    # Setup the Photo v1 API
-    SCOPES = 'https://www.googleapis.com/auth/photoslibrary https://www.googleapis.com/auth/photoslibrary.sharing'
     store = file.Storage((Path().parent / "auth/google_token.json").resolve().as_posix())
-    creds = store.get()
-    # creds = None
-    if not creds or creds.invalid:
-        flow = client.flow_from_clientsecrets((Path().parent / "auth/google_credentials.json").resolve().as_posix(), SCOPES)
-        creds = tools.run_flow(flow, store)
+    return store.get()
 
-    return creds
 
 def get_google_photos_service(google_creds):
-    return build('photoslibrary', 'v1', http=google_creds.authorize(Http()))
+    return build('photoslibrary', 'v1',
+                 http=google_creds.authorize(Http()),
+                 cache_discovery=False)
```

Comment:
no cache discovery, https://github.com/googleapis/google-api-python-client/issues/299, avoid the exceptions / warnings.